tests/accessibility/actions/preview-text-message-template.actions.js

```diff
@@ -0,0 +1,39 @@
+const {
+  pageActions: goToCreateTextMessageTemplateActions,
+} = require('./create-text-message-template.actions');
+
+const url = (baseUrl) => `${baseUrl}/create-template`;
+
+const pageActions = [
+  ...goToCreateTextMessageTemplateActions,
+  'set field #smsTemplateName to example-template-1',
+  'set field #smsTemplateMessage to example template message',
+  'click element #create-sms-template-submit-button',
+  'wait for #review-sms-template-submit-button to be visible',
+];
+
+const reviewTextMessageTemplatePage = (baseUrl) => ({
+  name: 'preview-sms-app-template',
+  url: url(baseUrl),
+  actions: pageActions,
+});
+
+const reviewTextMessageTemplateErrorPage = (baseUrl) => ({
+  name: 'preview-sms-app-template-error',
```

Comment:
change to `preview-sms-template-error`?

Reply:
changed in this commit -> https://github.com/NHSDigital/nhs-notify-web-template-management/pull/116/commits/08eb00f62862f81bddad787513a5a103a3994e6c